gearmand/datadog_checks/gearmand/gearmand.py

```diff
@@ -99,14 +99,13 @@ def _get_conf(self, instance):
     def check(self, instance):
         self.log.debug("Gearman check start")
 
-        host, port, task_filter, tags = self._get_conf(instance)
-        service_check_tags = ["server:{0}".format(host), "port:{0}".format(port)]
+        host, port, task_filter, instance_tags = self._get_conf(instance)
+
+        tags = ["server:{0}".format(host), "port:{0}".format(port)] + instance_tags
```

Comment:
Note that it's better to use extend, to not create a new list.

Reply:
Hum, IMHO this version is fine. this is a bit more atomic/functional, avoid changing the tags by extending.

But I'm fine with both, doesn't matter that much :)

Reply:
```bash
; python -m timeit '["aa", "bb"] + ["cc"]'
2000000 loops, best of 5: 99.2 nsec per loop
; python -m timeit '["aa", "bb"].extend(["cc"])'
2000000 loops, best of 5: 131 nsec per loop
```

Reply:
It's not about speed, but memory usage.